errors.md

```diff
@@ -4,6 +4,7 @@
 * BIBTEX MISSING abrahamson1965vocab
```

Comment:
@LanguageStructure, please check these errors carefully, as they point to some problems in the data and should be corrected before merging this PR.